build.sbt

```diff
@@ -35,25 +35,23 @@ releaseProcess := Seq[ReleaseStep](
 resolvers ++= Resolver.sonatypeOssRepos("releases")
 
 libraryDependencies ++= Seq(
-  "com.gu" %% "content-api-models-scala" % "25.0.0",
+  "com.gu" %% "content-api-models-scala" % "26.0.0",
   "com.gu" %% "thrift-serializer" % "5.0.5",
-  "software.amazon.kinesis" % "amazon-kinesis-client" % "2.6.0",
+  "software.amazon.kinesis" % "amazon-kinesis-client" % "3.0.1",
```

Comment:
I did not read this document about upgrading from KCL v2 to v3:

https://docs.aws.amazon.com/streams/latest/dev/kcl-migration-from-2-3.html

...this is unfortunate as it probably means that the release that included this PR ([v1.0.26](https://github.com/guardian/content-api-firehose-client/releases/tag/v1.0.26)) is probably missing [at least one crucial setting](https://docs.aws.amazon.com/streams/latest/dev/kcl-migration-from-2-3.html#kcl-migration-from-2-3-configuration), and so should be avoided.

At the very least, the increased IAM permissions required by KCL v3 means that https://github.com/guardian/apple-news/pull/398 failed:

* https://github.com/guardian/apple-news/pull/398#issuecomment-2596272378